data/applications-menu.appdata.xml.in

```diff
@@ -6,6 +6,14 @@
   <name>Applications Menu</name>
   <summary>Open and search for apps</summary>
   <releases>
+    <release version="2.11.0" date="2021-11-26" urgency="medium">
+      <description>
+      <p>New features:</p>
```

Comment:
I wouldn't call this a new feature as searching settings is already a feature.  It more of an improvement/fix in the way the search works.